src/Build/Construction/ProjectElementContainer.cs

```diff
@@ -757,6 +807,11 @@ System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator()
                 return _enumerator;
             }
 
+            bool ICollection<ProjectElement>.Remove(ProjectElement item)
```

Comment:
I do'nt know if style guide allows, but all these one liner methods could be expression bodied method syntax (`=>`)

The code was originally written before that language feature.